posthog/models/property.py

```diff
@@ -29,13 +41,26 @@
 NEGATED_OPERATORS = ["is_not", "not_icontains", "not_regex", "is_not_set"]
 CLICKHOUSE_ONLY_PROPERTY_TYPES = ["static-cohort", "precalculated-cohort"]
 
+UNIX_TIMESTAMP_IN_SECONDS = "^\\d{10}\\.\\d{3}$"
```

Comment:
Unix timestamp is always ten digits (at least for a long enough timescale for us). Our SDK sends it with fractions of a second and (as far as I can tell) that's always ten digits followed by three digits.

I guess it's fair that representing it as a regex (to avoid inventing a language to describe it) makes it seem like it might be used to match...

I'd assume the digits after the decimal point are past any resolution we care about... so we could treat any ten digit timestamp as equivalent to the same ten digit number with any decimal part. Then we could send "unix_timestamp" as the property and nobody has to mentally compile the regex